packages/material-ui/src/Snackbar/Snackbar.js

```diff
@@ -212,6 +212,8 @@ const Snackbar = React.forwardRef(function Snackbar(props, ref) {
         window.removeEventListener('blur', handlePause);
       };
     }
+
+    return () => {};
```

Comment:
The consistent return rule is really annoying for effect hooks. `return undefined` should work too which is a bit better since this looks like you forgot to add cleanup logic.

Does a bare `return;` work as well?

Reply:
> The consistent return rule is really annoying for effect hooks.

I know!

> Does a bare return; work as well?

Sadly no, but as you said `return undefined does`